src/mono/mono/mini/mini.h

```diff
@@ -2811,12 +2811,20 @@ void mono_cfg_add_try_hole (MonoCompile *cfg, MonoExceptionClause *clause, guint
 void mono_cfg_set_exception (MonoCompile *cfg, MonoExceptionType type);
 void mono_cfg_set_exception_invalid_program (MonoCompile *cfg, const char *msg);
 
+#if defined(HOST_WASM) && !(defined(DEBUG) || defined(_DEBUG))
```

Comment:
do we really set these in debug builds for mono?

I can see in mono.proj that we set them on windows sometimes, and also in android builds.  but not elsewhere.

Reply:
That's a great question, I don't know how to answer it

Reply:
@kg 

```c
# if defined(HOST_WASM) && !(defined (DEBUG) || defined(_DEBUG)
...
#elif defined(HOST_WASM)
# error "this shouldn't happen on Release lanes"
#else
...
#endif
```

and see which lanes fail to build?

Reply:
Might be better to just use #ifdef HOST_WASM, and enable it manually if somebody needs it.